packages/internal-workflow-common/src/activity-options.ts

```diff
@@ -87,6 +80,80 @@ Note that the Temporal Server doesn't detect Worker process failures directly. I
   cancellationType?: coresdk.workflow_commands.ActivityCancellationType;
 }
 
+/**
+ * Options for local activity invocation
+ */
+export interface LocalActivityOptions {
+  /**
+   * Identifier to use for tracking the activity in Workflow history.
+   * The `activityId` can be accessed by the activity function.
+   * Does not need to be unique.
+   *
+   * @default an incremental sequence number
+   */
+  activityId?: string;
```

Comment:
This doesn't exist for local activities in Java and Go (but if it has value in TS, cool)

Reply:
Hmmm.. doesn't look like it's recorded anywhere, might be worth removing this.
@Sushisource

Reply:
Removing.